### PR TITLE
Remove dependency on EPEL repo on CI

### PR DESCRIPTION
**Pull request by rtnpro**
- use centos-admin repo instead
- install python packages for jenkins-job-builder using pip inside virtualenv.

### CODE REVIEW

**Comment by rtnpro** [comment]
#dotests

File: provisions/group_vars/all [hunk 1]
```diff
@@ -7,3 +7,5 @@ cccp_source_repo: https://github.com/centos/container-pipeline-service
 cccp_source_branch: master
 
 public_registry: registry.centos.org
+
+enable_epel: true
```
Comment (navidshaikh) [inline provisions/group_vars/all]: should this be true ?

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

File: provisions/group_vars/all (hunk 1 above)
Comment (mohammedzee1000) [inline provisions/group_vars/all]: My guess would be the default value and the ci overrides the same

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Pull request merged**